lib/file-utilities.js

```diff
@@ -117,10 +117,12 @@ function streamFiles(config, logfiles, callback) {
       debug('tailing: ' + file)
       tail = spawn('tail', ['-n0', '-F', file])
       tail.stdout.on('data', (data) => {
-        data = data.toString().trim().split('\n')
-        for (i = 0; i < data.length; i++) {
-          data[i] = {t: Date.now(), l: data[i], f: file, label: labels}
-          linebuffer.addMessage(data[i])
+        const lines = data.toString().trim().split('\n')
+        for (i = 0; i < lines.length; i++) {
+          client.log.agentLog({
+            line: lines[i]
```

Comment:
the node-client will auto-add the timestamp

Reply:
yep.  There were many avenues to pass `timestamp`, so I'm letting the client do it.